numpy/core/setup.py

```diff
@@ -853,6 +853,7 @@ def get_mathlib_info(*args):
             join('src', 'multiarray', 'dragon4.c'),
             join('src', 'multiarray', 'dtype_transfer.c'),
             join('src', 'multiarray', 'einsum.c.src'),
+            join('src', 'multiarray', 'einsum.dispatch.c.src'),
```

Comment:
Headers need to be listed in this file too

Reply:
since `einsum_helpers.h ` is a private header only used by  `einsum.c.src` and `einsum.dispatch.c.src`

Reply:
The fact it's private isn't relevant from what I remember: headers should be in the list above so that rebuilds can detect they've changed. See my PR for details.

Reply:
thank you for the clarification